### PR TITLE
RTIR integration: allow updating the MemberOf/Members

### PR DESCRIPTION
**Pull request by moishce**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-dc.paloaltonetworks.com/browse/CIAC-9146)

## Description
allow updating the MemberOf/Members in create/update commands

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/RTIR/ReleaseNotes/1_0_20.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### RTIR
+
+- added *memebers*, *member-of* arguments to **rtir-create-ticket** and  **rtir-edit-ticket** commands to updating the MemberOf and HasMembers fields.
```
Comment (ShahafBenYakir) [inline Packs/RTIR/ReleaseNotes/1_0_20.md:5]:
```suggestion
- added the *members* and *member-of* arguments to **rtir-create-ticket** and  **rtir-edit-ticket** commands to allow setting of the MemberOf and HasMembers fields.
```

File: Packs/RTIR/Integrations/RTIR/RTIR.yml [hunk 2]
```diff
@@ -90,6 +90,12 @@ script:
       name: initial-priority
     - description: Ticket final priority, given as number in the range 0-100.
       name: final-priority
+    - description: Ticket MemberOF links.
```
Comment (ShahafBenYakir) [inline Packs/RTIR/Integrations/RTIR/RTIR.yml:93]: What about depends-on and refers-to? I see they appear in the desigm

**Review by ShahafBenYakir — COMMENTED** [review]
In the design doc I see that the enhcnament was not meant to be added to the *rtir-create-ticket* command. Why have we add this?

**Review by ShahafBenYakir — COMMENTED** [review]
Good job, see pre-commit right comment.

File: Packs/RTIR/Integrations/RTIR/RTIR.yml [hunk 3]
```diff
@@ -90,6 +90,24 @@ script:
...
+      isArray: true
+      name: member-of
+    - description: Ticket Members links.
```
Comment (ShahafBenYakir) [inline Packs/RTIR/Integrations/RTIR/RTIR.yml:96]: The input should be a link? i.e. a url? not an ID?

Comment (moishce) [reply Packs/RTIR/Integrations/RTIR/RTIR.yml:96]: ticket number/id

Comment (ShahafBenYakir) [reply Packs/RTIR/Integrations/RTIR/RTIR.yml:96]: in that case let's fic the description it says `Ticket MemberOF links.` which is confusing, probably needs to be `Ticket MemberOF ID` etc...

**Review by ShahafBenYakir — APPROVED** [review]

**Comment by ShahafBenYakir** [comment]
Just fix the descriptions

**Pull request merged**